detox/android/detox/src/testFull/java/com/wix/detox/espresso/action/DetoxMultiTapSpec.kt

```diff
@@ -136,17 +137,17 @@ object DetoxMultiTapSpec: Spek({
         }
 
         it("should throw if no UI-controller provided") {
-            assertFailsWith(KotlinNullPointerException::class) {
+            assertFailsWith(NullPointerException::class) {
```

Comment:
@jonathanmos What's that about?

Reply:
https://blog.jetbrains.com/kotlin/2019/08/kotlin-1-3-50-released/#npe-optimizations